src/utils/date.ts

```diff
@@ -74,3 +74,27 @@ export const formatDate = (date: Date, long = false) => {
 
   return `${date.toLocaleString(undefined, options)}`
 }
+
+type UnitType = 'week' | 'day' | 'hour' | 'minute' | 'second'
+// in miliseconds
+const units: Record<UnitType, number> = {
+  week: 7 * 24 * 60 * 60 * 1000,
+  day: 24 * 60 * 60 * 1000,
+  hour: 60 * 60 * 1000,
+  minute: 60 * 1000,
+  second: 1000,
+}
+
+var rtf = new Intl.RelativeTimeFormat('en', { numeric: 'always' })
```

Comment:
Really cool that this is supported natively 🚀

Reply:
I also learned it through the requirement :D